src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -239,6 +251,81 @@ InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name)
     }
 }
 
+InferenceEngine::Parameter MKLDNNExecNetwork::GetMetric(const std::string &name) const {
+    if (_graphs.empty())
+        IE_THROW() << "No graph was found";
+    // @todo Can't we just use local copy (_cfg) instead?
+    auto graphLock = GetGraph();
+    const auto& graph = graphLock._graph;
+    const auto& config = graph.getProperty();
+
+    if (isLegacyAPI()) {
+        return GetMetricLegacy(name, graph);
+    }
+
+    auto RO_property = [](const std::string& propertyName) {
+        return ov::PropertyName(propertyName, ov::PropertyMutability::RO);
+    };
+
+    if (name == ov::supported_properties) {
+        return std::vector<ov::PropertyName> {
+            RO_property(ov::model_name.name()),
+            RO_property(ov::optimal_number_of_infer_requests.name()),
+            RO_property(ov::streams::num.name()),
+            RO_property(ov::affinity.name()),
+            RO_property(ov::inference_num_threads.name()),
+            RO_property(ov::enable_profiling.name()),
+            RO_property(ov::hint::inference_precision.name()),
+            RO_property(ov::hint::performance_mode.name()),
+            RO_property(ov::hint::num_requests.name()),
+        };
+    }
+
+    if (name == ov::model_name) {
+        // @todo Does not seem ok to 'dump()' the whole graph everytime in order to get a name
+        return graph.dump()->get_friendly_name();
+    } else if (name == ov::optimal_number_of_infer_requests) {
+        const auto streams = config.streamExecutorConfig._streams;
+        return static_cast<uint32_t>(streams); // ov::optimal_number_of_infer_requests has no negative values
+    } else if (name == ov::streams::num) {
+        const auto streams = config.streamExecutorConfig._streams;
+        return static_cast<int32_t>(streams); // ov::streams::num has special negative values (AUTO = -1, NUMA = -2)
+    } else if (name == ov::affinity) {
+        const auto affinity = config.streamExecutorConfig._threadBindingType;
+        switch (affinity) {
+        case InferenceEngine::IStreamsExecutor::ThreadBindingType::NONE:
+            return ov::Affinity::NONE;
+        case InferenceEngine::IStreamsExecutor::ThreadBindingType::CORES:
+            return ov::Affinity::CORE;
+        case InferenceEngine::IStreamsExecutor::ThreadBindingType::NUMA:
+            return ov::Affinity::NUMA;
+        case InferenceEngine::IStreamsExecutor::ThreadBindingType::HYBRID_AWARE:
+            return ov::Affinity::HYBRID_AWARE;
+        }
+        return ov::Affinity::NONE;
+    } else if (name == ov::inference_num_threads) {
+        const auto num_threads = config.streamExecutorConfig._threads;
+        return num_threads;
+    } else if (name == ov::enable_profiling.name()) {
+        const bool perfCount = config.collectPerfCounters;
+        switch (perfCount) {
+        case true: return "YES";
+        case false: return "NO";
```

Comment:
@ilya-lavrenov 
Because there is a python API which still uses get_config / GetConfig() methods directly to get parameters
we have to return YES / NO instead of bool even for new API
since there is no auto-conversion made in this case.